src/pages/ProductDetail/DetailTop/Options/Options.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { ImLocation } from 'react-icons/im';
+
+function Options({ item }) {
+  const [option, setOption] = useState({});
+
+  return (
+    <Container>
+      <OptionsWrap>
+        <span>{item.title}</span>
+        {item.title === '상품상태' ? (
+          <p>{option.condition}중고</p>
+        ) : item.title === '교환여부' ? (
+          <p>{option.exchangable}교환불가능</p>
+        ) : item.title === '거래지역' ? (
+          <p>
+            <ImLocation />
+            {option.region}
+            전국
+          </p>
+        ) : (
+          <p>
+            {option.createDate}
+            1분 전 등록
+          </p>
+        )}
```

Comment:
```suggestion
   {itemOption[item.title] ? (
          <p>
            {item.title === '거래지역' && <ImLocation />}
            {option[itemOption[item.title]]}
            {itemText[itemOption[item.title]]}
          </p>
        ) : (
          <p>
            {option.createDate}
            1분 전 등록
          </p>
        )}
```

위의 리뷰를 토대로 위와 같이 변경할 수 있겠네요!